x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/OptimizerVerificationTests.java

```diff
@@ -393,10 +393,7 @@ public void testRemoteEnrichAfterLookupJoinWithPipelineBreaker() {
             | LOOKUP JOIN languages_lookup ON language_code
             | ENRICH _remote:languages ON language_code
             """, analyzer);
-        assertThat(
-            err,
-            containsString("4:3: LOOKUP JOIN with remote indices can't be executed after [STATS c = COUNT(*) by languages]@2:3")
-        );
+        assertThat(err, containsString("5:3: ENRICH with remote policy can't be executed after [STATS c = COUNT(*) by languages]@2:3"));
```

Comment:
Why did this change? That join should be marked as remote, but looks like it wasn't. That doesn't feel right?

Reply:
@smalyshev I think it makes sense for how current implementation works (though we should probably make the [docs](https://www.elastic.co/docs/reference/query-languages/esql/esql-cross-clusters#esql-enrich-remote) reflect the actual behavior).

I restored the old behavior, now the ENRICH is considered remote, and the LOOKUP is validated accordingly.